rsc/client/opendota/match_integration_test.go

```diff
@@ -0,0 +1,44 @@
+//go:build integration
+// +build integration
+
+package opendota
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFetchMatchDetail(t *testing.T) {
+	ctx := context.Background()
+	t.Run("fetch match detail dota2 should be success", func(t *testing.T) {
+		openDotaAPIURL := "https://api.opendota.com/api"
+		httpClient := http.DefaultClient
+
+		openDotaAPI := NewMatchAPI(httpClient, openDotaAPIURL)
```

Comment:
same comment as https://github.com/ronaudinho/drt/pull/8#discussion_r1185919812, since these variables are reused in the other test, maybe declare them on parent function (like `ctx`)